config/database.yml.sample

```diff
@@ -9,8 +9,10 @@ default: &default
   encoding: unicode
   pool: <%= ENV.fetch("RAILS_MAX_THREADS") { 5 } %>
   timeout: 5000
-  host: localhost
-  username: <%= `echo $USER` %>
+  username: postgres
+  password: password
+  host: db
```

Comment:
the values here should be env variables and fallback to strings so it works both with and without docker